ui/app/AppLayouts/Chat/controls/community/AirdropTokensSelector.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import utils 1.0
+
+StatusFlowSelector {
+    id: root
+
+    property alias model: repeater.model
+
+    signal itemClicked(int index, var mouse)
+
+    placeholderText: qsTr("Example: 1 SOCK")
+    placeholderItem.visible: repeater.count === 0
+
+    title: qsTr("What")
+    icon: Style.svg("token")
+
+    QtObject {
+        id: d
+
+        readonly property int commonMargin: 6
+        readonly property int iconSize: 28
+    }
+
+    Repeater {
+        id: repeater
+
+        Control {
+            component Icon: StatusRoundedImage {
+                implicitWidth: d.iconSize
+                implicitHeight: d.iconSize
+            }
+
+            component Text: StatusBaseText {
+                Layout.fillWidth: true
+
+                color: Theme.palette.primaryColor1
+                elide: Text.ElideRight
+            }
+
+            implicitHeight: root.placeholderItemHeight
+            leftPadding: (root.placeholderItemHeight - d.iconSize) / 2
+            rightPadding: d.commonMargin * 2
+
+            background: Rectangle {
+                color: Theme.palette.primaryColor3
+                radius: root.placeholderItemHeight / 2
+
+                MouseArea {
+                    anchors.fill: parent
+                    cursorShape: Qt.PointingHandCursor
+                    onClicked: root.itemClicked(model.index, mouse)
+                }
+            }
+
+            contentItem: RowLayout {
```

Comment:
Also as discussed offline I think we can have benefits of abstracting it to a generic component as a `StautsDoubleTagItem` or similar.

Reply:
StatusTagItem with `leftIcon` and `rightIcon` props, respectively?

Reply:
I was thinking about that but customization would be probably longer than this implementation here, imo it's not worth to complicate things too much